sympy/series/tests/test_limits.py

```diff
@@ -615,6 +615,12 @@ def test_issue_8481():
     assert limit(lamda**k * exp(-lamda) / factorial(k), k, oo) == 0
 
 
+def test_issue_8634():
+    n = Symbol('n', integer=True, positive=True)
+    x = Symbol('x')
+    assert limit(x**n, x, -oo) == oo/(-1)**n
```

Comment:
The result looks odd. Maybe there should be `(-1)**-ex` on line 326 above.

Reply:
if I remember correctly it used to be like that originally but Christopher  had told me to change the form , for easy interpretation of a test on one of my prs (#22491)
So it was me who had changed this , surely could be changed back but couple tests would have to be changed I reckon !

Reply:
I had tried a decent amount of examples on terminal before making the commit .The result was consistent with forms of `oo/(-1)**n` .I'm not too sure how to change the result ( I believed the above one was expected ) ,so I would request some assistance for that !

Reply:
I reckon we could try changing line 326 to `S.NegativeInfinity*sign(coeff)*S.NegativeOne**(S.One + ex)` and change tests based on the current implementation of line 326 to what we obtain from the new implementation , @faze-geek you could try that !

Reply:
Okay I'll make the change in a commit soon ( assuming this is inline with what Kalevi suggested )